test/tools/isoltest.cpp

```diff
@@ -115,6 +115,11 @@ class TestTool
 
 	Result process();
 
+	void printOptions(
+		ostream& _stream,
+		string const& _linePrefix = ""
+	) const;
```

Comment:
This is short enough to be on a single line now.
```suggestion
	void printOptions(ostream& _stream, string const& _linePrefix = "") const;
```